packages/formatter-html/src/assets/js/scan/_locales/en/messages.js

```diff
@@ -0,0 +1,122 @@
+// Autogenerated by scripts/compile-locales.js
+/* eslint-disable quotes, quote-props */
+(function () {
```

Comment:
Just to clarify, do these particular generated files have to be committed (e.g. the scanner reads them from `src`)? Or can the scanner read these from the published `npm` package so we only need to generate them on build (and can omit them from `git`)?

Reply:
For now, we need these generated files to be commited because we are using them when generating the website. These files will not be necessary once https://github.com/webhintio/webhint.io/issues/757 is done.

There is an issue to track this: #2722.